locale/fi/toast.json

```diff
@@ -0,0 +1,50 @@
+{
+  "title": "Ilmoitus (Toast)",
+  "intro": "Ilmoitus on komponentti, jonka avulla käyttäjälle voidaan välittää tietoa käyttäjän toimintaan liittyvää palautetta. Käytetään tilanteissa, jotka eivät vaadi käyttäjältä varsinaista reagointia. Toast-ilmoitus kertoo, että käyttäjän suorittama toimenpide onnistui.",
+  "note.title": "Saavutettavuus ja käytettävyys",
+  "note.items": [
+    "Mikäli ilmoituksia on tarkoitus lisätä sivulle dynaamisesti, niitä varten kannattaa luoda ruudunlukijakäyttäjiä varten live region(ulkoinen linkki sivun alaraunasssa), jonka sisään ilmoitukset lisätään. Tällöin myös ruudunlukijakäyttäjät saavat tiedon sivulle lisätystä ilmoituksesta.",
+    "Ilmoituksen otsikkotaso on määriteltävissä käyttötapauksen mukaan.",
+    "Sivulla voi olla yhtä aikaa useita Toast-ilmoituksia näkyvissä."
+  ],
+  "sections": [
+    {
+      "title": "Mistä komponentti koostuu",
+      "paragraphs": [
+        {
+          "image.key": "",
+          "image.alt": "",
+          "text": "Komponentti koostuu kuvaavasta ikonista, otsikosta ja pääsisällöstä. Komponentissa ei ole eri tiloja (status variant)."
+        }
+      ],
+      "links": [
+        {
+          "text": "",
+          "url": ""
+        }
+      ]
+    },
+    {
+      "title": "Koko ja käyttö",
+      "paragraphs": [
+        {
+          "image.key": "",
+          "image.alt": "",
+          "text": "Toast-ilmoitukset pysyvät näkyvissä joko 10 sekunttia tai niin kauan kuin käyttäjä pitää hiirtä ilmoituksen päällä. Ilmoitukset poistuvat näkyvistä liukumalla ylös."
+        },
+        {
+          "image.key": "",
+          "image.alt": "",
+          "text": "Desktopissa, kun sivu on scrollattu ylös, ilmoitus näkyy sivun ylälaitaan kiinnittyneenä. Kun käyttäjä scrollaa sivulla alaspäin niin, että sivu ylälaita peittyy, liukuu ilmoitus näkyviin oikeaan ylälaitaan. Ilmoitus siis kelluu mukana, kun sivua scrollataan alas."
+        }
+      ],
+      "links": [
+        {
+          "text": "Live region",
+          "url": "https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Live_Regions"
+        }
+      ]
+    }
+  ],
+  "exampleRegular.content": "Tallentaminen onnistui"
```

Comment:
Maybe the key could be simplified as there is no possibility to conflict with different kind of examples here.
Maybe just `example.content` (changing it here and in other .json files of course)

Reply:
changed to "example.content"

Reply:
Could you change the value in the **af/toast.json** also, now it has the old keyname in it.
https://github.com/vrk-kpa/suomifi-design-system/blob/0bc89c22d852c7e3b32b69016680df7e6ab1981e/locale/af/toast.json#L45

Reply:
fixed.